src/XMLimport.cpp

```diff
@@ -922,6 +923,7 @@ void XMLimport::readHostPackage( Host * pT )
     pT->mUSE_FORCE_LF_AFTER_PROMPT = ( attributes().value("mUSE_FORCE_LF_AFTER_PROMPT") == "yes" );
     pT->mUSE_UNIX_EOL = ( attributes().value("mUSE_UNIX_EOL") == "yes" );
     pT->mNoAntiAlias = ( attributes().value("mNoAntiAlias") == "yes" );
+    pT->mEchoLuaErrors = ( attributes().value("mEchoLuaErrors") == "yes" );
```

Comment:
Presumably `attributes().value("mEchoLuaErrors")` returns a default constructed empty `QString` when the attribute is not present in files pre-dating the addition of this feature so that it will default to __OFF__ if the user's setup has never encountered this option before - which is what I'd expect.